packages/expo/src/web/Image/AssetSourceResolver.ts

```diff
@@ -0,0 +1,117 @@
+import { Platform } from 'expo-core';
+import invariant from 'invariant';
+import { Dimensions } from 'react-native';
+
+import {
+  getAndroidResourceFolderName,
```

Comment:
Remove all this Android stuff -- keep just what's needed for web

Reply:
There's maybe very little used on web if we remove all the iOS & Android stuff -> the bundle size can be smaller + we have less surface area to maintain